index.html

```diff
@@ -229,7 +229,80 @@ <h1>The Property Registry</h1>
     <p>
       The following section defines the properties in a DID document.
     </p>
+    
     <section>
+      <h2><dfn data-lt="@context" data-dfn-type="dfn" id="@context">@context</dfn></h2>
+      <p>Required for JSON-LD. Optional for representations other than JSON-LD.</p>
+      <pre class="example">
+        {
+          "@context": [
+            "https://w3id.org/did/v0.11",
```

Comment:
I hesitate to provide URLs that work, but are going to be wrong in time. The spec says the URL should be https://www.w3.org/ns/did/v1 ... I get what you're going for (something that works if people follow the registry). The right thing to do here is ensure that v0.11 and v1 are the same, and then use the https://www.w3.org/ns/did/v1 URL above.

Reply:
https://www.w3.org/ns/did/v1 is not stable until the DID WG decides it is right? there will be no way to keep these aligned I think.

I think its probably wise for us to not reference w3id.org whenever we can.

Reply:
> https://www.w3.org/ns/did/v1 is not stable until the DID WG decides it is right? there will be no way to keep these aligned I think.

Well, we can request that @iherman push the latest versions from time to time, or create some sort of auto-publishing process (maybe)... or do a 301 redirect until the spec hits CR? There are mechanisms that we can use to keep it more aligned than it is now, and I think we should use those.

> I think its probably wise for us to not reference w3id.org whenever we can.

For the DID WG Contexts, that's fine... but for the vocabulary, we'll break all the implementations in terrible ways if we change those w3id.org URLs, so let's not do that.